pkgs/by-name/mi/minbrowser/package.nix

```
@@ -0,0 +1,78 @@
+{
+  lib,
+  stdenv,
+  fetchurl,
+
+  dpkg,
+  autoPatchelfHook,
+
+  libxkbcommon,
+  libxcb,
+  xorg,
+  alsa-lib,
+  nss,
+  at-spi2-core,
+  mesa,
+  cairo,
+  pango,
+  cups,
+  gtk3,
+  glib,
+
+  nix-update-script,
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "min";
+  version = "1.34.1";
+
+  src = fetchurl {
+    url = "https://github.com/minbrowser/min/releases/download/v${finalAttrs.version}/min-${finalAttrs.version}-amd64.deb";
+    hash = "sha256-gpkjGYuHwBY3IwK5bXhzIPPosSTZ67hclmGLT4PTsG4=";
+  };
+
+  nativeBuildInputs = [
+    dpkg
+    autoPatchelfHook
+  ];
+
+  buildInputs = [
+    libxkbcommon # libxkbcommon.so
+    libxcb # libxcb.so
+    at-spi2-core # libatspi.so
+    mesa # libdbm.so libexpat.so
```

```suggestion
    libgbm # libdbm.so libexpat.so
```
Unless one of those libraries is not in libgbm